lib/embed.dart

```diff
@@ -449,6 +449,21 @@ class Embed {
     return isLegalGistId(id) ? id : '';
   }
 
+  // A unique string for this sample in the context of a codelab or article
+  String get snippetId {
+    final url = Uri.parse(window.location.toString());
+    if (!url.queryParameters.containsKey('snippetid')) {
+      return null;
+    }
+
+    final value = url.queryParameters['snippetid'];
```

Comment:
It's pedantic, but "snippetId" might not be the best name for the query param ("snippet" is used for a specific thing in the API Doc server, and there might be other potential collisions elsewhere). If this is just the string that we're using to identify something for GA, consider something specific to that, like "ga_token" or "ga_id".

Also nomenclature-related: I've been using snake case for multi-word query params (`sample_id` and the various `gh_whatever` params, for instance). Unless you feel strongly, I'd prefer to keep that pattern for any new params.

Reply:
fixed - changed to gh_id